src/tree/VirtualMachineTreeItem.ts

```diff
@@ -15,6 +15,12 @@ import { nonNullProp, nonNullValueAndProp } from '../utils/nonNull';
 import { treeUtils } from '../utils/treeUtils';
 
 export class VirtualMachineTreeItem extends AzureTreeItem {
+    public static runningContextValue: RegExp = /^VirtualMachine.+running/;
+    public static stoppedContextValue: RegExp = /^VirtualMachine.+stopped/;
+    public static linuxContextValue: RegExp = /^VirtualMachine.+linux/;
+    public static windowsContextValue: RegExp = /^VirtualMachine.+windows/;
+    public static allContextValue: RegExp = /^VirtualMachine/;
```

Comment:
I noodled on how to do these better (I believe that's what you were asking for help on) but I think they look pretty clean. The best I came up with was being a little more specific with the regex but I don't think it's required:

```ts
    public static runningContextValue: RegExp = /^VirtualMachine-(linux|windows)-running$/;
    public static stoppedContextValue: RegExp = /^VirtualMachine-(linux|windows)-stopped$/;
    public static allContextValue: RegExp = /^VirtualMachine-(linux|windows)-(running|stopped)$/;
```

Reply:
> I think I could use some help here... I think that there should be an easier way to just construct the RegEx for the tree picker than manually typing everything out

It's a pain and I have not found a clean way to do this. I feel like we might be able to stick some shared logic in the UI package to establish a consistent pattern and make this easier for all repos, but I haven't gotten around to it. Y'all are more than welcome to take a stab at it if you want!